doc/users/project/mission.rst

```diff
@@ -0,0 +1,27 @@
+Matplotlib Mission Statement
+============================
+
+The mission of the Matplotlib developer community is to develop, maintain, and
```

Comment:
Maybe split in two sentences: Matplotlib is… The Mission of the Matplotlib community is …

Reply:
@timhoffm do you like Jody's edits?